meinberlin/templates/a4dashboard/includes/progress_line_in_progress.html

```diff
@@ -69,7 +83,7 @@
         {% if project.is_draft and project_progress.valid != project_progress.required or project.published_modules.count == 0 %}
```

Comment:
I tried to correct this one but my brain is sad, for projects, this doesn't become filled dot when a module is added to project, i'll have another look in morning

Reply:
But this should only become filled when it's possible to publish the project? Oh, yeah, this is not working when a module is created, but not filled in, even if another module was added to the project. It's the same for the publish button, I think. Will ignore for now and have a think later!